tests/integration/test_ses.py

```diff
@@ -49,3 +57,32 @@ def test_get_identity_verification_attributes(self):
             self.assertEqual('Success', response[value]['VerificationStatus'])
         self.assertIn('VerificationToken', response[domain])
         self.assertNotIn('VerificationToken', response[email])
+
+    def test_send_email_save(self):
+        client = aws_stack.connect_to_service('ses')
+        data_dir = config.DATA_DIR
+        if data_dir == '':
+            data_dir = config.TMP_FOLDER
+        email = 'user@example.com'
+        client.verify_email_address(EmailAddress=email)
+        message = client.send_email(Source=email, Message={
+            'Subject': {
+                'Data': 'A_SUBJECT',
+            },
+            'Body': {
+                'Text': {
+                    'Data': 'A_MESSAGE',
+                },
+            },
+        }, Destination={
+            'ToAddresses': ['success@example.com'],
+        })
+
+        f = open(os.path.join(data_dir, 'ses', message['MessageId'] + '.json'), 'r')
```

Comment:
nit: Same a above - we could use `with open(..) ...` here, instead of calling `.close()` directly.